components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -55,6 +55,9 @@ interface PanelHandler {
   // Notify the backend that the dialog should be closed.
   CloseUI();
 
+  AddEthereumChainApproved(string network, int32 tab_id);
+  AddEthereumChainCanceled(string EthereumChain, int32 tab_id);
```

Comment:
Could you avoid using both camel case and snake case in your changes in this file?
I know we're inconsistent in this file, but could you use just snake case in your changes here?
It's quite weird to see both in this line here.